tests/testthat/_snaps/dm_to_tibble.md

```diff
@@ -0,0 +1,458 @@
+# dm_to_tibble and friends work
+
+    Code
+      print(tbl_keys_from_dm_tf_4$tbl)
+    Output
+      # A tibble: 5 x 6
+        h     i     j        j1 tf_5             tf_3$g $tf_2           
+        <chr> <chr> <chr> <int> <list>           <chr>  <list>          
+      1 a     three C         3 <tibble [0 x 3]> two    <tibble [0 x 3]>
+      2 b     four  D         4 <tibble [1 x 3]> three  <tibble [1 x 3]>
+      3 c     five  E         5 <tibble [1 x 3]> four   <tibble [2 x 3]>
+      4 d     six   F         6 <tibble [1 x 3]> five   <tibble [2 x 3]>
+      5 e     seven F         6 <tibble [1 x 3]> five   <tibble [2 x 3]>
+    Code
+      print(tbl_keys_from_dm_tf_4$tbl$tf_3$tf_2)
+    Output
+      [[1]]
+      # A tibble: 0 x 3
+      # ... with 3 variables: c <chr>, d <int>, tf_1 <tibble[,1]>
+      
+      [[2]]
+      # A tibble: 1 x 3
+        c            d tf_1$b
+        <chr>    <int> <chr> 
+      1 elephant     2 B     
+      
+      [[3]]
+      # A tibble: 2 x 3
+        c         d tf_1$b
+        <chr> <int> <chr> 
+      1 lion      3 C     
+      2 dog       6 F     
+      
+      [[4]]
+      # A tibble: 2 x 3
+        c         d tf_1$b
+        <chr> <int> <chr> 
+      1 seal      4 D     
+      2 cat       7 G     
+      
+      [[5]]
+      # A tibble: 2 x 3
+        c         d tf_1$b
+        <chr> <int> <chr> 
+      1 seal      4 D     
+      2 cat       7 G     
+      
+    Code
+      print(tbl_keys_from_dm_tf_4$tbl$tf_5)
+    Output
+      [[1]]
+      # A tibble: 0 x 3
+      # ... with 3 variables: k <int>, m <chr>, tf_6 <tibble[,1]>
+      
+      [[2]]
+      # A tibble: 1 x 3
+            k m     tf_6$o
+        <int> <chr> <chr> 
+      1     1 house e     
+      
+      [[3]]
+      # A tibble: 1 x 3
+            k m     tf_6$o
+        <int> <chr> <chr> 
+      1     2 tree  f     
+      
+      [[4]]
+      # A tibble: 1 x 3
+            k m          tf_6$o
+        <int> <chr>      <chr> 
+      1     3 streetlamp h     
+      
+      [[5]]
+      # A tibble: 1 x 3
+            k m          tf_6$o
+        <int> <chr>      <chr> 
+      1     4 streetlamp h     
+      
+
+---
+
+    Code
+      print(serialized_from_tf_4)
+    Output
+      # A tibble: 5 x 6
+        h     i     j        j1 tf_5_json_child             tf_3_json_parent          
```

Comment:
Should we keep one column per parent or child table also in serialized form?

Reply:
I don't understand